windows/ReactTestApp/MainPage.h

```diff
@@ -12,8 +12,9 @@ namespace winrt::ReactTestApp::implementation
                               Windows::UI::Xaml::RoutedEventArgs);
         void LoadFromDevServer(Windows::Foundation::IInspectable const &,
                                Windows::UI::Xaml::RoutedEventArgs);
-        winrt::Windows::Foundation::IAsyncAction
-            OnNavigatedTo(Windows::UI::Xaml::Navigation::NavigationEventArgs);
+        Windows::Foundation::IAsyncAction
+        OnNavigatedTo(Windows::UI::Xaml::Navigation::NavigationEventArgs const &e);
+        using base = MainPageT;
```

Comment:
Nit: We should use `Base` since it's really a class name.
```suggestion
        using Base = MainPageT;
```